api/src/test/java/uk/gov/dhsc/htbhf/eligibility/EligibilityServiceIntegrationTests.java

```diff
@@ -65,6 +67,7 @@ void shouldReturnEligibleResponseGivenEligibleResponseReturnedFromDwp() throws J
         verifyDWPEndpointCalled();
     }
 
+    @Disabled("nino is optional for private beta and unique key is yet to be decided")
     @Test
```

Comment:
It looks like this was testing whether a bad request is returned using nino, instead of nino can we use another mandatory field and check whether bad request is still returned?

Reply:
yes, changed to surname to test bad request